machines/auth.ts

```diff
@@ -127,11 +147,25 @@ export const authMachine = model.createMachine(
       setBiometrics: model.assign({
         biometrics: (_, event: SetupBiometricsEvent) => event.biometrics,
       }),
+
+      setLanguage: assign({
+        selectLanguage: (context) => !context.selectLanguage,
+      }),
     },
 
     services: {
-      downloadFaceSdkModel: () => () => {
-        downloadModel();
+      downloadFaceSdkModel: () => async () => {
+        var injiProp = null;
```

Comment:
These code block is already present in commonProps.ts, please use downloadModel method, instead of duplicating the code block.

Reply:
Used downloadModal as a method imported from commonProps.ts to auth.ts.